bucket/plex-desktop.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.29.1.1974-a65b3ef3",
+    "description": "Plex for Windows.",
+    "homepage": "https://www.plex.tv/apps-devices/#modal-devices-plex-for-windows",
+    "license": "Proprietary",
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.plex.tv/plex-desktop/1.29.1.1974-a65b3ef3/windows/Plex-1.29.1.1974-a65b3ef3-x86_64.exe#/dl.7z",
+            "hash": "sha1:7aa89b4f525094136c3025deb6b684fb2e51a56c"
+        }
+    },
+    "shortcuts": [
+        [
+            "Plex.exe",
+            "Plex for Windows"
+        ]
+    ],
+    "checkver": {
+        "url": "https://plex.tv/pms/downloads/6.json",
+        "jsonpath": "$.computer.Windows.version"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://downloads.plex.tv/plex-desktop/$version/windows/Plex-$version-x86_64.exe#/dl.7z",
+                "hash": {
+                    "url": "https://plex.tv/pms/downloads/6.json",
+                    "jsonpath": "$.computer.Windows.releases.checksum"
```

Comment:
Add condition for build `windows-x86_64`

Reply:
i've rebase it witht he suggestion + a new version that was released